changelog/pending/20230128--backend-filestate--the-filestate-backend-now-supports-project-scoped-stacks.yaml

```diff
@@ -0,0 +1,6 @@
+changes:
+- type: feat
+  scope: backend/filestate
+  description: |
+    The filestate backend now supports project scoped stacks.
+    Newly initialized storage will automatically use this mode.
```

Comment:
Since this is going to show up in the changelog, you might as well include the new environment variables for visibility.

Reply:
Added the opt-out, but not the one to silence the warning since that's mentioned as part of the warning message.
(And it has less meaning without explaining in depth what mixed state is, or how you might end up there.)

Do you think we need the other one in here too?

Reply:
I feel like publishing new ENV VARS in the changelog is generally good.

Reply:
Good point. Everything added to changelogs.